MAUI/DataForm/localization.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: post
+title: Data localization in .NET MAUI DataForm control | Syncfusion
+description: Learn about the data localization support in Syncfusion .NET MAUI DataForm(SfDataForm) control in mobile and desktop applications from a single shared codebase.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Localization in .NET MAUI DataForm (SfDataForm)
+
+Localization is the process of translating the application resources into different language for the specific cultures. The `SfDataForm` can be localized by adding `resource` file. 
+
+You can localize the `DataFormViewItem` [DisplayAttribute](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute?view=net-7.0) values and validation attributes ([Required](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.requiredattribute?view=netframework-4.8), [StringLength](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.stringlengthattribute?view=netframework-4.8)) values by using `ResourceType` property of the attribute or using the `GenerateDataFormItem` event when autogenerate true.
+
+## Setting CurrentUICulture to the application
+
+Application culture can be changed by setting `CurrentUICulture` in `App.xaml.cs` file.
+
+{% tabs %}
+{% highlight C# tabtitle="App.xaml.cs" %}
+
+using System.Globalization;
+
+public partial class App : Application
+{
+	public App()
+	{
+		InitializeComponent();
+
+		MainPage = new AppShell();
+        var culture = new CultureInfo("fr");
+        CultureInfo.CurrentUICulture = culture;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+N>
+The required `resx` files with `Build Action` as `EmbeddedResource` (File name should contain culture code) into the `Resources` folder.
+
+## Add Localization resx file in sample level
+
+To localize the `DataForm` based on `CurrentUICulture` using `resource` files, follow the below steps.
```

Comment:
Follow these steps to localize the `DataForm` based on the `CurrentUICulture` using the `resource` files.

Reply:
content changed